articles/connect-end-user-to-wifi-with-certificate.md

```diff
@@ -0,0 +1,307 @@
+# Connect end users to Wi-Fi or VPN with a certificate (DigiCert, NDES, or custom SCEP)
+
+_Available in Fleet Premium_
+
+Fleet can help your end users connect to Wi-Fi or VPN by deploying certificates from your certificate authority (CA). Fleet currently supports [DigiCert](https://www.digicert.com/digicert-one), [Microsoft NDES](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/network-device-enrollment-service-overview), and custom [SCEP](https://en.wikipedia.org/wiki/Simple_Certificate_Enrollment_Protocol) server.
+
+
+## DigiCert
+
+To connect end users to W-Fi or VPN with DigiCert certificates, we'll do the following steps:
+
+- [Create service user in DigiCert](#step-1-create-service-user-in-digicert)
+- [Create certificate profile in DigiCert](#step-2-create-certificate-profile-in-digicert)
+- [Connect Fleet to DigiCert](#step-3-connect-fleet-to-digicert)
+- [Add PKCS #12 configuration profile to Fleet](#step-4-add-pkcs-12-configuration-profile-to-fleet)
+
+### Step 1: Create service user in DigiCert
```

Comment:
@lukeheath @georgekarrv, @marko-lisica and I learned that we can't test this step because service users aren't available in our DigiCert demo instance. 

Do y'all think we (Fleet) can get a production DigiCert instance as a partner? This way, we can test the best practice workflow.